interface/templates/interface/search.html

```diff
@@ -18,13 +18,13 @@
     <div class="ui text menu">
         <div class="ui item">
             <div class="ui large breadcrumb">
-                {% include 'interface/breadcrumbs.html' with page_name=repo.full_name %}
+                {% include 'interface/breadcrumbs.html' %}
             </div>
         </div>
-        {% include 'interface/user_dropdown.html' with private=repo.is_private %}
```

Comment:
Rather than removing the `with`s (because as I've said they are necessary), just change the values sent.
I suggest `page_name='Search'` and `private=False`.